proto/one/operations/sample/test_analyte_group.proto

```diff
@@ -28,8 +28,8 @@ message TestAnalyteGroup {
     // Soft Delete of the test analyte group
     bool isActive = 7;
 
-    // Collection of analytes in the test analyte group
-    repeated Analyte analytes = 8;
+    // Collection of identifiers of each Analyte in this Test Group
+    repeated string AnalyteIds = 8;
```

Comment:
I think the property name should be analyteIds with a lowercase first letter to match the others.

Reply:
Nice catch, fixed it, thanks